app/controllers/admin/recent_activity_collections_controller.rb

```diff
@@ -13,11 +13,11 @@ def index
     private
 
     def days_limit
-      params.fetch(:admin_recent_activity, {}).fetch(:days_limit, 7).to_i
+      recent_activity_params.fetch(:days_limit, Admin::RecentActivityForm.new.days_limit).to_i
```

Comment:
the default value of "7" is provided in the form, let's use it here (else if you change in one spot and not the other it won't work as expected)